src/emucore/Settings.cxx

```diff
@@ -300,6 +300,8 @@ void Settings::usage() {
        "     Phosphor blends screens to reduce flicker\n"
        "   -record_screen_dir [save_directory]\n"
        "     Saves game screen images to save_directory\n"
+       "   -stochasticity (default: 0.25)\n"
```

Comment:
I'll change it.

On 25 May 2015 at 05:33, Marc G. Bellemare <notifications@github.com> wrote:

> In src/emucore/Settings.cxx
> <https://github.com/mgbellemare/Arcade-Learning-Environment/pull/55#discussion_r30975514>
> :
>
> > @@ -300,6 +300,8 @@ void Settings::usage() {
> >         "     Phosphor blends screens to reduce flicker\n"
> >         "   -record_screen_dir [save_directory]\n"
> >         "     Saves game screen images to save_directory\n"
> > +       "   -stochasticity (default: 0.25)\n"
>
> Can we name this 'repeat_action_probability' instead? We know what
> "stochasticity" means because we've been talking about it, but it will
> likely be very cryptic for anyone else.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mgbellemare/Arcade-Learning-Environment/pull/55/files#r30975514>
> .
>



-- 
Marlos Cholodovskis Machado
http://www.dcc.ufmg.br/~marlos <http://www.cs.ualberta.ca/~machado>